add_dimension.json

```diff
@@ -50,7 +50,7 @@
         }
     ],
     "returns": {
-        "description": "The data cube with a newly added dimension.",
+        "description": "The data cube with a newly added dimension. The new dimension has exactly one dimension label. All other dimensions have not changed.",
```

Comment:
> All other dimensions have not changed.

Maybe it's better to check with a native English speaker, but the past tense here in a `returns` doc sounds bit strange to me. Maybe it's easier to just say "are unchanged" or "do not change"

Reply:
> All other dimensions have not changed.

> "are unchanged" or "do not change"

What about:
All other dimensions remain unchanged.

Reply:
That would work in this case, but there are more phrases used in the past tense, so we need to clarify the tense we want to use anyway.